### DISCUSSION TITLE
WebRTC Provider

### DISCUSSION THREAD

**Discussion by getroot** [OvenMediaLabs/OvenMediaEngine#1078] [Announcement]
The stability of the WebRTC Provider has been greatly improved and the Screen Sharing function has been reopened accordingly. You can test it with https://demo.ovenplayer.com/demo_input.html and the latest OvenMediaEngine master branch.

There have been many reports of problems with WebRTC input. It's difficult to comment on all related issues, so I'm writing this as a new issue. Please comment if you find any related issues.

Note that encoding may be required when WebRTC input -> LLHLS output. This is because WebRTC inputs change resolution frequently, but some LLHLS players do not handle resolution changes. LLHLS is better suited for ABR than dynamic resolution.

**Comment by bchah** [comment]
@getroot Thank you for this! Doing a quick test of the dev build this morning and there is a *huge* difference in the quality and consistency of the WebRTC provider. Playback starts faster and the quality ramps up faster too. Excellent work, thank you again.
